src/main/java/com/wcc/platform/controller/ProgrammeController.java

```diff
@@ -29,7 +29,13 @@ public ProgrammeController(final ProgrammeService programmeService) {
   @Operation(summary = "API to retrieve programme page")
   @ResponseStatus(HttpStatus.OK)
   public ResponseEntity<ProgrammePage> getProgramme(
-      @RequestParam(name = "type") final ProgramType programmeType) {
-    return ResponseEntity.ok(service.getProgramme(programmeType));
+      @RequestParam(name = "type") final String programmeType) {
+    final ProgramType programType = ProgramType.fromString(programmeType);
+
+    if (ProgramType.BOOK_CLUB.equals(programType)) {
```

Comment:
Can you add document why are we hardcoding ProgramType to be BOOK_CLUB. 
Will we extend the same api for other ProgramTypes in future.